biome.jsonc

```diff
@@ -58,20 +50,14 @@
             "deniedGlobals": ["fetch"]
           }
         },
-        // We prefer to use `Math.pow` over `**` operator
-        "useExponentiationOperator": "off",
```

Comment:
what's the reason that we have some rules here and some in `@chainsafe/biomejs-config`?

Reply:
I gotta be honest I am not a big fan of having some rules here and some in the config in some other repo, but not strongly opposed either

I brought this up here https://github.com/ChainSafe/lodestar/issues/6011#issuecomment-1929088462 before, but seems like @wemeetagain is in favor of using a module as well

Reply:
Well I'm not really in favor of having a massive diff between the upstream config and this config. The idea was to push as much as possible upstream and force-standardize a lodestar way of doing things everywhere.

Upstream can and should be opinionated. Its not industry best practices, its our practices, for better or worse.
If we have a huge diff here, then its kinda worst of both worlds. Lack of flexibility and also lack of consistency.

Reply:
That said,
if there are some rules that need to be retained here temporarily (eg: "TODO: we want to enable/disable this rule but it requires some refactor")
or if there really are circumstances specific to this repo or a subdirectory in the repo (eg: "This file does things differently for XYZ reasons"),
those are valid reasons for extending the config... so imo it's not completely black and white.